digitaltwin_client/inc/digitaltwin_device_client.h

```diff
@@ -51,10 +51,11 @@ typedef void* DIGITALTWIN_DEVICE_CLIENT_HANDLE;
   @param    deviceHandle[in]            An IOTHUB_DEVICE_CLIENT_HANDLE that has been already created and bound to a specific connection string (or transport, or DPS handle, or whatever
                                         mechanism is preferred).  See remarks about its lifetime management.
   @param    dtDeviceClientHandle[out]   A <c>DIGITALTWIN_DEVICE_CLIENT_HANDLE</c> to be used by the application.
- 
+  @param    deviceCapabilityModel[in]   The device's Device Capability Model (DCM) identifier. 
+
   @returns  A DIGITALTWIN_CLIENT_RESULT.
 */
-MOCKABLE_FUNCTION(, DIGITALTWIN_CLIENT_RESULT, DigitalTwin_DeviceClient_CreateFromDeviceHandle, IOTHUB_DEVICE_CLIENT_HANDLE, deviceHandle, DIGITALTWIN_DEVICE_CLIENT_HANDLE*, dtDeviceClientHandle);
+MOCKABLE_FUNCTION(, DIGITALTWIN_CLIENT_RESULT, DigitalTwin_DeviceClient_CreateFromDeviceHandle, IOTHUB_DEVICE_CLIENT_HANDLE, deviceHandle, DIGITALTWIN_DEVICE_CLIENT_HANDLE*, dtDeviceClientHandle, const char*, deviceCapabilityModel);
```

Comment:
>const char*, deviceCapabilityModel [](start = 196, length = 34)

The deviceCapabilityModel as an IN parameter should (by convention rest of SDK uses) come before the dtDeviceClientHandle, which is the OUT param.  Similar feedback for the _LL_ version of this function.